src/pages/Admin/Room/Edit/RoomEditModal.tsx

```diff
@@ -0,0 +1,194 @@
+      className="room-edit-modal"
+    >
+      <Modal.Header>
+        <Modal.Title>Editing - {room.title}</Modal.Title>
+      </Modal.Header>
+
+      <Modal.Body>
+        <Form onSubmit={handleSubmit(onSubmit)}>
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Name your room</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="title"
+                placeholder="Room name"
+                custom
+              />
+              {errors.title && (
+                <span className="input-error">{errors.title.message}</span>
+              )}
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>The room url</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="url"
+                placeholder="Room url"
+                custom
+              />
+              {errors.url && (
+                <span className="input-error">{errors.url.message}</span>
+              )}
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>The room description (optional)</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="description"
+                placeholder="Description"
+                custom
+              />
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>
+                How you want the room to appear on the map
+              </Form.Label>
+              <ImageInput
+                onChange={handleImageChange}
+                name="image"
+                forwardRef={register}
+                small
+                nameWithUnderscore
+                imgUrl={room.image_url}
+              />
+              {errors.image_url && (
+                <span className="input-error">{errors.image_url.message}</span>
+              )}
+            </div>
+          </Form.Row>
+
+          {customInputFields}
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Change room template</Form.Label>
+              <Form.Control
+                as="select"
+                custom
+                name="template"
+                ref={register}
+                onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
+                  setSelectedTemplate(e.target.value)
+                }
+              >
+                {ROOM_TEMPLATES.map((template) => (
+                  <option key={template.name}>{template.template}</option>
+                ))}
+              </Form.Control>
+            </div>
+          </Form.Row>
```

Comment:
Was this removed in the stuff in `staging`? I believe we wanted to remove it in general, but not sure if we got around to that yet.

Probably [OOS] if we haven't removed it already. But mostly want to make sure we aren't 'resurrecting' anything here that we didn't want to keep (so probably do a 'visual spot check' between the `RoomEdit` file in `staging` currently + this one to ensure they're aligned.

Reply:
It wasn't removed in staging (I can open a new pr after this to remove it)